docs/static/breaking-changes.asciidoc

```diff
@@ -34,6 +34,17 @@ These changes can impact any instance of Logstash and are plugin agnostic, but o
 * Configurations provided with `-f` or `config.path` will not be appended with `stdin` input and `stdout` output automatically.
 
 [float]
+
+==== Elasticsearch output changes
+
+* The default `document_type` used has changed from `logs` to `doc`. 
```

Comment:
You can remove "used" and simply say: The default `document_type` has changed from `logs` to `doc`.